models/behavior.js

```diff
@@ -0,0 +1,9 @@
+class Behavior {
```

Comment:
@ryzokuken Can we rename this class into something like `BehaviorModel` since there is also a class with same name in `/behaviors/index.js` ?